<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="paramTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>661</width>
      <height>321</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>EDT [s]</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>T20 [s]</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>T30 [s]</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>C50 [dB]</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>C80 [dB]</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Tt [s]</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>EDTTt [s]</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>INR [dB]</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>IACCe</string>
     </property>
    </row>
   </widget>
   <widget class="PlotWidget" name="graphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>661</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setupButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyze another file</string>
    </property>
   </widget>
   <widget class="QListWidget" name="channelList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>650</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Export data...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
